src/main/java/io/mosip/mimoto/util/Utilities.java

```diff
@@ -194,6 +194,11 @@ public static ResponseWrapper<Object> handleExceptionWithErrorCode(Exception exc
             errorCode = errorSections[0];
             errorMessage = errorSections[1];
         }
-        return getErrorResponse(errorCode, errorMessage);
+        return new String[]{errorCode, errorMessage};
+    }
+
+    public static List<ErrorDTO> getErrors(String errorCode, String errorMessage) {
```

Comment:
who is using this method?

Reply:
Its been used to create a errors object in multiple places.